config/locales/en.yml

```
@@ -19,6 +19,7 @@ en:
   label_extension: Extension
   label_text_extraction: Text extraction
   label_identifier: ID
+  label_label: Tags
```

`label_wiki_ext_tag` とかだと少し違和感がありますかね
`label_tag` にしたいけど、[ActionView::Helpers::FormTagHelper#label_tag](https://api.rubyonrails.org/classes/ActionView/Helpers/FormTagHelper.html#method-i-label_tag)と被ってしまうのが悩ましいですねmm
```suggestion
  label_wiki_ext_tag: Tags
```

---

https://github.com/clear-code/redmine_full_text_search/pull/137/files#diff-749f9eeb9a08c198d06c317bf7a82650ebcb7399e3963120715d486263885d94R31
この名前と合わせなくていけなくて、そこが「label」なので「label_label」になりましたが、たしかに微妙な感じはあります。

「label」にした経緯は以下の通りです。

* タグって他のプラグインでも使ってそうだし、汎用的に「tag」みたいな名前にしたい
  * ので、専用を示す「wiki_ext_tag」みたいなのは避けたい
* けど、fts_tagsの中に「tag」ってあると紛らわしいので、汎用的ないい感じの名前にしたい
  *  「label」とかよさそう？（他に良いのがあればそれにしたい）

---

なるほどです。
`label` 良さそうに思います。実際に Rails 側でも `tag_label` という命名のメソッドがあるため

---

ラベルにしました。